src/pages/selectJourneyDirection.tsx

```diff
@@ -3,6 +3,7 @@ import { NextPageContext } from 'next';
 import { parseCookies } from 'nookies';
 import Layout from '../layout/Layout';
 import { OPERATOR_COOKIE, SERVICE_COOKIE, JOURNEY_COOKIE } from '../constants';
+import { deleteCookieOnServerSide } from '../utils';
```

Comment:
Should we make this page `returnDirection` and the other direction page `singleDirection` to make it a bit clearer?